docs/rule/no-invalid-role.md

```diff
@@ -0,0 +1,41 @@
+## no-invalid-role
+
+This rule checks for invalid element/role combinations.
+
+Current list of checks:
+
+1. Use of the presentation role for content which should convey semantic information may prevent the user from understanding that content. This rule checks semantic HTML elements for the presence of `role="none"` or `role="presentation"`. Elements that are permitted to explicitly have these role values are `<div>`, `<img>`, `<span>` and `<svg>`.
```

Comment:
the last sentence is a little misleading. we use a blacklist of elements in the implementation, but this description makes it sound like a whitelist. that had be confused because I was wondering if it worked for `IndustryStandardCase` component invocations or not 😉

Reply:
that's my fault, I flipped the implementation but missed this. I'll update, thank you for catching it!